content/en/cloud_cost_management/anomalies.md

```diff
@@ -28,7 +30,7 @@ On the [Anomalies tab of the Cloud Cost page in Datadog][1], you can view the an
 - **Past**: If an anomaly lasts more than 7 days, or the algorithm detects that costs are no longer anomalous, the anomaly is moved to the **Past** tab. Past anomalies can be useful to report on, but are often less urgent and actionable.
 - **Resolved**: If you [resolve an anomaly](#resolve-anomalies), the anomaly can be found in the **Resolved** tab.
 
-Each anomaly explains how much more costs were than expected for the service name (ex:`rds`), usage type, and cloud accounts for the anomaly. The anomaly also shows what expected costs would have been in the time frame. The card also shows a graph with the cost trend over the past 1 month.
+Each anomaly explains how much more costs were than expected for the service name (ex:'rds'), usage type, and cloud accounts for the anomaly. The anomaly also shows what expected costs would have been in the time frame. The card also shows a graph with the cost trend over the past 1 month.
```

Comment:
```suggestion
Each anomaly explains how much costs exceeded expectations for the service name (ex:'rds'), usage type, and cloud accounts. Additionally, the anomaly also shows what expected costs would have been in the time frame. The anomaly card shows a graph with the cost trend over the past 1 month.
```

Reply:
accepting part of this - "exceeded expectations" isn't the right wording here (that makes it sound more positive than it is)